ament_copyright/ament_copyright/__init__.py

```diff
@@ -1,3 +1,17 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Actually, it looks like GitHub changes the source for the generation of their LICENSE file for apache 2.0, so they may have changed the indention under our feet between us creating the two repositories:

https://github.com/github/choosealicense.com/commit/d2b1145e6d1a89a74caa7a8594a8d671c75828d1#diff-772b67594a65caad6cdd48c76b29f9a2R227

I actually noticed this before I think on one of my personal projects.